service/src/main/java/uk/gov/hmcts/reform/fpl/utils/EmailNotificationHelper.java

```diff
@@ -37,18 +37,23 @@ public static String buildSubjectLine(final CaseData caseData) {
 
     public static String buildSubjectLineWithHearingBookingDateSuffix(final String subjectLine,
                                                                       final List<Element<HearingBooking>>
-                                                                          hearingDetails) {
+                                                                          hearingBookings) {
         String hearingDate = "";
-        if (isNotEmpty(hearingDetails)) {
-            hearingDate = " hearing " + dateFormatterService.formatLocalDateToString(
-                hearingBookingService.getMostUrgentHearingBooking(
-                    hearingDetails).getStartDate().toLocalDate(), FormatStyle.MEDIUM);
+        if (isNotEmpty(hearingBookings)) {
```

Comment:
Not convinced we need this, there's not going to be a case where hearingBookings are missing. What do you think?